app.js

```diff
@@ -106,22 +120,46 @@ function displayFriends() {
         friendEl.addEventListener('click', () => {
             // > handle the three possible outcomes:
             // 1. No mushrooms, set a message to go hunt for more
-            // 2. Friend is already fully satisfied (3), set a message to pick another friend
-            // 3. Feed friend mushroom:
-            // a. "pop" a mushroom off the mushrooms array
-            // b. increase friend.satisfied by 1
-            // c. set a message that the friend enjoyed the mushroom,
-            //    include the friend name and mushroom type in the message
-
+            if (mushrooms.length === 0) {
```

Comment:
since you want this ```if``` check to evaluate to true you could also do ```if (!mushrooms.length)``` — if you don't have any mushrooms, your ```mushrooms``` array is empty, the length of an empty array is ```0```, ```0``` evaluates to ```false``` (it's a falsy value so not equal to ```false``` (```0 !== false```)), and ```!``` makes it the opposite so it's ```true``` when there are no mushrooms